### PR TITLE
Document `exception_to_status` on resources and operations

### PR DESCRIPTION
**Pull request by julienfalque**
Documentation for https://github.com/api-platform/core/pull/3519.

### CODE REVIEW

File: core/errors.md [hunk 1]
```diff
@@ -100,3 +100,35 @@ the error will be returned in this format as well:
   "hydra:description": "The product \"1234\" does not exist."
 }
 ```
+
+## Fine-grained Configuration
+
+Since version 2.7, the `exception_to_status` configuration can be set on resources and operations:
```
Comment (alanpoulain) [inline core/errors.md:106]:
```suggestion
The `exception_to_status` configuration can be set on resources and operations:
```

File: core/errors.md [hunk 2]
```diff
@@ -100,3 +100,35 @@ the error will be returned in this format as well:
...
+
+```php
+use ApiPlatform\Core\Annotation\ApiResource;
```
Comment (alanpoulain) [inline core/errors.md:109]: Usually we start with the `<?php` tag.

File: core/errors.md [hunk 3]
```diff
@@ -100,3 +100,35 @@ the error will be returned in this format as well:
...
+/**
+ * @ApiResource(
+ *     itemOperations={
+ *         "get"={
+ *             "exception_to_status"={
+ *                 "App\Exception\ProductWasRemovedException": 410
+ *             }
+ *         }
+ *     },
+ *     attributes={
```
Comment (alanpoulain) [inline core/errors.md:120]: Wouldn't it better to use a root-level attribute instead (`exceptionToStatus`) instead?
Your core PR needs to be changed too.

File: core/errors.md [hunk 4]
```diff
@@ -100,3 +100,35 @@ the error will be returned in this format as well:
...
+
+## Fine-grained Configuration
+
+Since version 2.7, the `exception_to_status` configuration can be set on resources and operations:
+
+```php
+use ApiPlatform\Core\Annotation\ApiResource;
+
+/**
+ * @ApiResource(
```
[the remark this answers is not in the document]
Comment (julienfalque) [reply core/errors.md]: Done.

File: core/errors.md (hunk 1 above)
Comment (julienfalque) [reply core/errors.md]: Updated.

File: core/errors.md (hunk 2 above)
Comment (julienfalque) [reply core/errors.md:113]: Updated.

File: core/errors.md (hunk 3 above)
Comment (julienfalque) [reply core/errors.md]: Agreed. Do you want me to do it or do you prefer doing it? Should using `attributes` be supported anyway?

Comment (alanpoulain) [reply core/errors.md]: It would be nice if you could do it :slightly_smiling_face:
It is supported mainly for backward compatibility.

Comment (julienfalque) [reply core/errors.md]: > It is supported mainly for backward compatibility.

You mean `attributes` is?

Comment (alanpoulain) [reply core/errors.md]: Yes! It's not really deprecated but I think it's better to promote the root-level way to do it.

File: core/errors.md [hunk 5]
```diff
@@ -100,3 +100,37 @@ the error will be returned in this format as well:
...
+
+#[ApiResource(
+    itemOperations: [
+        'get' => [
+            'exception_to_status' => [
+                ProductWasRemovedException::class => 410,
+            ],
+        ],
+    ],
+    exception_to_status: [
```
Comment (alanpoulain) [inline core/errors.md:125]:
```suggestion
    exceptionToStatus: [
```

Comment (julienfalque) [reply core/errors.md]: Fixed.

**Comment by alanpoulain** [comment]
Thank you!

**Pull request merged**